tests/testMatcherSourceSelector.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+#
+# Copyright 2008-2016  AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import division, absolute_import, print_function
+from builtins import range
+
+import unittest
+import numpy as np
+
+import lsst.afw.table as afwTable
+from lsst.meas.algorithms import sourceSelector
+import lsst.meas.base.tests
+import lsst.utils.tests
+
+
+badFlags = [
```

Comment:
Hey here's the current text I've written. Let me know if this is what you want and I'll commit it.

"
# badFlags are flags that, if the bad flag is set the objecct should be
# imediately thrown out. This is as opposed to combinations of flags
# signifying good or bad data. Such flags should then be a part of the
# isGood test
"